src/OrchardCore.Modules/OrchardCore.Settings/Views/Settings.Edit.cshtml

```diff
@@ -18,31 +18,49 @@
+        <span class="hint">@T["Determines the default time zone used when displaying and editing dates and times. DST (daylight saving time) will be applied automatically per time zone if available."]</span>
+    </fieldset>
+</div>
+
+<h5 class="form-group">@T["Resource Settings"]<span class="hint"> — @T["Settings for scripts and stylesheets"]</span></h5>
 <fieldset class="form-group" asp-validation-class-for="AppendVersion">
     <div class="custom-control custom-checkbox">
         <input type="checkbox" class="custom-control-input" asp-for="@Model.AppendVersion" />
         <label class="custom-control-label" asp-for="@Model.AppendVersion">@T["Use resources cache busting"]</label>
-        <span class="hint">@T[" — Whether cache busting (append a version) is used for scripts and stylesheets"]</span>
+        <span class="hint">— @T["Whether cache busting (append a version) is used for scripts and stylesheets"]</span>
     </div>
 </fieldset>
 
 <fieldset class="form-group" asp-validation-class-for="UseCdn">
     <div class="custom-control custom-checkbox">
         <input type="checkbox" class="custom-control-input" asp-for="@Model.UseCdn" />
-        <label class="custom-control-label" asp-for="@Model.UseCdn">@T["Use CDN (Content Delivery Network)"]</label>
-        <span class="hint">@T[" — Whether the defined CDN value is used for scripts and stylesheets, or their local version"]</span>
+        <label class="custom-control-label" asp-for="@Model.UseCdn">@T["Use framework CDN (Content Delivery Network)"]</label>
+        <span class="hint">— @T["Whether a framework CDN is used for registered scripts and stylesheets, such as jQuery, or their local version"]</span>
     </div>
 </fieldset>
 
 <fieldset class="form-group" asp-validation-class-for="CdnBaseUrl">
-    <label asp-for="CdnBaseUrl">@T["CDN Base url"]</label>
+    <label asp-for="CdnBaseUrl">@T["Site CDN (Content Delivery Network) base url"]</label>
     <input asp-for="CdnBaseUrl" class="form-control" />
     <span asp-validation-for="CdnBaseUrl"></span>
-    <span class="hint">@T["A CDN Base url applied to locally hosted scripts and stylesheets, if empty no CDN Base url is applied, and is disabled in Resource Debug Mode"]</span>
+    <span class="hint">@T["A base CDN URL prefixed to the sites, scripts and stylesheets, if empty no base CDN URL is applied, and it is disabled in Resource Debug Mode"]</span>
 </fieldset>
 
-<div class="row">
-    <fieldset class="from-group col-xl-6" asp-validation-class-for="ResourceDebugMode">
+<div class="row form-group">
+    <fieldset class="col-xl-6" asp-validation-class-for="ResourceDebugMode">
         <label asp-for="ResourceDebugMode">@T["Resource Debug Mode"]</label>
         <select asp-for="ResourceDebugMode" class="form-control">
             <option value="@ResourceDebugMode.FromConfiguration.ToString()">@T["From environment — enabled in <em>Production</em>, disabled otherwise)"]</option>
```

Comment:
```suggestion
            <option value="@ResourceDebugMode.FromConfiguration.ToString()">@T["From environment — enabled in <em>Production</em>, disabled otherwise"]</option>
```